Dockerfile

```diff
@@ -53,7 +53,7 @@ USER root
 # For testing
 RUN microdnf update -y \
 && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-1.8.0-openjdk-devel \
-&& microdnf install -y procps gzip unzip tar shadow-utils findutils util-linux less rsync git \
+&& microdnf install -y procps gzip unzip tar shadow-utils findutils util-linux less rsync git which \
```

Comment:
You probably want to all `which` to the other two Docker files.

Reply:
> You probably want to all `which` to the other two Docker files.

Absolutely :+1: . The complete solution will need `which` installed in each Dockerfile... but this PR is just demonstrating what needs to be done for the main/single docker image :smile: